db/seeds.rb

```diff
@@ -1,8 +1,8 @@
 require 'nokogiri'
 require 'open-uri'
 
-user = User.create(email: 'test@gmail.com',
-                   password: '123456')
+user = User.create(name: 'Test', email: 'test@gmail.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.